test/system/070-build.bats

```diff
@@ -620,6 +620,14 @@ cat >$tmpdir/.dockerignore <<EOF
 *
 EOF
 
+    # Make sure that building a local pause image ignores ignore-files (#13529)
+    pushd "${tmpdir}"
+    run_podman pod create
+    run_podman pod rm $output
+    local pause=$(pause_image)
+    run_podman rmi $pause
```

Comment:
Thanks! No real need for a variable, though: could just be `run_podman rmi $(pause_image)` like you did below.